app/models/post.rb

```diff
@@ -10,4 +10,27 @@ def comments_counter
   def likes_counter
     likes.count
   end
+
+  def update_posts_counter
+    author.update(posts_count: author.posts_count + 1)
+  end
```

Comment:
- [ ] You can use one of the [Rails Active Callbacks](https://guides.rubyonrails.org/active_record_callbacks.html) - `after_save` to call the `#update_posts_counter` method every time a new post has been saved to database.